libraries/common/io/files.effekt

```diff
@@ -95,7 +95,7 @@ def readFile(path: String): String / { AsyncIO, Exception[IOError] } = {
 
   val readSize = 1048576 // 1MB
   var size = readSize
-  var buffer = bytes::empty(size)
+  var buffer = bytes::allocate(size)
```

Comment:
Should this be `allocate` or `bytes`? There is no other constructor here.

Reply:
It's closest to `array::allocate` in that it's unsafe.

Reply:
It's safe, no? It just initializes with 0s ;)

Reply:
I thought about this for a bit and you're right, I've renamed it to just `bytes`.